tests/sles4sap/saptune/mr_test.pm

```diff
@@ -47,8 +47,8 @@ sub setup {
     select_serial_terminal;
     # Disable packagekit
     quit_packagekit;
-    # saptune is not installed by default on SLES4SAP 12 on ppc64le and in textmode profile
-    zypper_call "-n in saptune" if ((is_ppc64le() and is_sle('<15')) or check_var('DESKTOP', 'textmode'));
```

Comment:
If there are still tests running for other architectures, it might be good to do a VR there just to be sure the change does not break anything.

Reply:
I guess it will still work as below those lines there is a `zypper_call "-n in saptune"` without a conditional.

What I don't like about this change is that it will obscure possible bugs when the `saptune` package is not installed by default as part of the `sap_server` pattern, i.e., installing `saptune` always will guarantee the SUT has `saptune` _for_ the test, but we won't quickly see whether `saptune` was installed by the test explicitly or if it was installed as part of the SLES for SAP Applications installation.

There used to be a test scenario using the `sles4sap/patterns` and `sles4sap/saptune` test modules that would cover this, by checking that saptune gets installed as part of the pattern, but I see it disabled on 15-SP5. It's still enabled for QAM.

So TL;DR: LGTM as long as we're covering elsewhere that `saptune` gets installed by default as part of the SLES for SAP Applications product.

Reply:
Thanks for the comments. Good to know. @alvarocarvajald 
And here is the VR for sles4sap 12sp5: https://openqaworker15.qa.suse.cz/tests/130986#step/mr_test/17 @lpalovsky